[Feedback] Page: Update Windows install instructions on page: /install/latest/self-hosted/installation-source/

### Is it easy to find the information you need?

No

### Are the instructions clear?

No

### How could we improve the Timescale documentation site?

There is no information about why this error occurs:
C:\Development\othergit\timescaledb>bootstrap.bat
-- Building for: Visual Studio 17 2022
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19042.
-- The C compiler identification is MSVC 19.33.31629.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.33.31629/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- TimescaleDB version 2.5.1. Can be updated from version 2.5.0
-- Build type is Release
-- Install method is 'source'
-- Using compiler MSVC
CMake Error at CMakeLists.txt:292 (message):
  Unable to find 'pg_config'

I assume pg_config is meant to be the full path to the postgres config file but nothing has said to set it and Postgres install should not be expected to set it because clearly it hasn't. Postgres itself is running successfully for me. It's not something silly like I haven't installed Postgres yet

Even when I create environment variable pg_config and set it to C:\Program Files\PostgreSQL\14\bin\ (as described in a sibling page of yours), the same error occurs

C:\Development\othergit\timescaledb>echo %pg_config%
C:\Program Files\PostgreSQL\14\bin\

C:\Development\othergit\timescaledb>bootstrap
Build system already initialized in ./build
Do you want to remove it (this is IMMEDIATE and PERMANENT), y/n? y
-- Building for: Visual Studio 17 2022
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19042.
-- The C compiler identification is MSVC 19.33.31629.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.33.31629/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- TimescaleDB version 2.5.1. Can be updated from version 2.5.0
-- Build type is Release
-- Install method is 'source'
-- Using compiler MSVC
CMake Error at CMakeLists.txt:292 (message):
  Unable to find 'pg_config'


-- Configuring incomplete, errors occurred!
See also "C:/Development/othergit/timescaledb/build/CMakeFiles/CMakeOutput.log".
---
TimescaleDB build system initialized in ./build.
To compile, do:
    cmake --build ./build --config Release

C:\Development\othergit\timescaledb>


**Ok, so the bin folder has to be in PATH. Don't expect Postgres install to do it, because it doesn't, so please mention that step.**

**You might also want to mention that the command line has to be run in administrator mode:**
...
  exec.c
     Creating library C:/Development/othergit/timescaledb/build/tsl/src/Release/timescaledb-tsl-2.5.1.lib and object C:/Development/othergit/timescaledb/build/tsl/src/Release/timescaledb-tsl-2.5.1.exp
  timescaledb-tsl.vcxproj -> C:\Development\othergit\timescaledb\build\tsl\src\Release\timescaledb-tsl-2.5.1.dll
  Building Custom Rule C:/Development/othergit/timescaledb/CMakeLists.txt

C:\Development\othergit\timescaledb>cmake --build ./build --config Release --target install
MSBuild version 17.3.1+2badb37d1 for .NET Framework
  timescaledb.vcxproj -> C:\Development\othergit\timescaledb\build\src\Release\timescaledb-2.5.1.dll
  timescaledb-loader.vcxproj -> C:\Development\othergit\timescaledb\build\src\loader\Release\timescaledb.dll
  timescaledb-tsl.vcxproj -> C:\Development\othergit\timescaledb\build\tsl\src\Release\timescaledb-tsl-2.5.1.dll
  -- Install configuration: "Release"
  -- Installing: C:/PROGRA~1/POSTGR~1/14/share/extension/timescaledb.control
  CMake Error at cmake_install.cmake:44 (file):
    file INSTALL cannot copy file
    "C:/Development/othergit/timescaledb/build/timescaledb.control" to
    "C:/PROGRA~1/POSTGR~1/14/share/extension/timescaledb.control": Permission
    denied.


C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: The command "setlocal [C:\Development\othergit\timescaledb\build\install.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake
\CMake\bin\cmake.exe" -DBUILD_TYPE=Release -P cmake_install.cmake [C:\Development\othergit\timescaledb\build\install.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd [C:\Development\othergit\timescaledb\build\install.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: :cmEnd [C:\Development\othergit\timescaledb\build\install.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone [C:\Development\othergit\timescaledb\build\i
nstall.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: :cmErrorLevel [C:\Development\othergit\timescaledb\build\install.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: exit /b %1 [C:\Development\othergit\timescaledb\build\install.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: :cmDone [C:\Development\othergit\timescaledb\build\install.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd [C:\Development\othergit\timescaledb\build\install.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(159,5): error MSB3073: :VCEnd" exited with code 1. [C:\Development\othergit\timescaledb\build\install.vcxproj]

--- a/install/installation-source.md
+++ b/install/installation-source.md
@@ -18,15 +18,18 @@ Before you start, make sure you have installed:
 *   PostgreSQL 12 or later, with a development environment. For more information
     about PostgreSQL installation, including downloads and instructions, see the
     [PostgreSQL documentation][postgres-download].
-*   CMake version 3.11 or later. For more information about CMake installation,
-    including downloads and instructions, see the
-    [CMake documentation][cmake-download].
+
+You also need:
+
+*   CMake version 3.11 or later for your operating system. For more information
+    about CMake installation, including downloads and instructions, see the [CMake documentation][cmake-download].
 *   C language compiler for your operating system, such as `gcc` or `clang`.
 
-If you are installing from source on a Microsoft Windows system, you also need:
-
-*   Visual Studio 2015 or later, packaged with CMake version 3.11 or later, and
-    Git components.
+<highlight type="note">
+If you are using a Microsoft Windows system, you can install Visual Studio 2015
+or later instead of CMake and a C language compiler. Ensure you install the
+Visual Studio components for CMake and Git when you run the installer.
+</highlight>
 
 <procedure>
 
@@ -35,7 +38,7 @@ If you are installing from source on a Microsoft Windows system, you also need:
 1.  At the command prompt, clone the Timescale GitHub repository:
 
     ```bash
-    git clone https://github.com/timescale/timescaledb.git
+    git clone https://github.com/timescale/timescaledb
     ```
 
 1.  Change into the cloned directory:
@@ -56,6 +59,7 @@ If you are installing from source on a Microsoft Windows system, you also need:
     not a branch. Continue with the steps in this procedure as normal.
 
 1.  Bootstrap the build system:
+
     <terminal>
 
     <tab label='Linux'>
@@ -75,7 +79,17 @@ If you are installing from source on a Microsoft Windows system, you also need:
     </tab>
 
     </terminal>
-1.  Build the extension:
+
+    <highlight type="important">
+    For installation on Microsoft Windows, you might need to add the `pg_config`
+    and `cmake` file locations to your path. In the Windows Search tool, search
+    for `system environment variables`. The path for `pg_config` should be
+    `C:\Program Files\PostgreSQL\<version>\bin`. The path for `cmake` is within
+    the Visual Studio directory.
+    </highlight>
+
+2.  Build the extension:
+
     <terminal>
 
     <tab label='Linux'>
@@ -95,7 +109,9 @@ If you are installing from source on a Microsoft Windows system, you also need:
     </tab>
 
     </terminal>
-1.  Install TimescaleDB:
+
+3.  Install TimescaleDB:
+
     <terminal>
 
     <tab label='Linux'>

--- a/install/installation-source.md
+++ b/install/installation-source.md
@@ -88,7 +88,7 @@ Visual Studio components for CMake and Git when you run the installer.
     the Visual Studio directory.
     </highlight>
 
-2.  Build the extension:
+1.  Build the extension:
 
     <terminal>
 
@@ -110,7 +110,7 @@ Visual Studio components for CMake and Git when you run the installer.
 
     </terminal>
 
-3.  Install TimescaleDB:
+1.  Install TimescaleDB:
 
     <terminal>
 

--- a/install/installation-windows.md
+++ b/install/installation-windows.md
@@ -40,16 +40,17 @@ current PostgreSQL installation, do not install TimescaleDB using this method.
 1.  Download and install the Visual C++ Redistributable for Visual Studio from
     [www.microsoft.com][ms-download].
 1.  Download and install PostgreSQL from [www.postgresql.org][pg-download].
-1.  In the Windows Search tool, search for `system environment variables`. In
-    the `System Properties` dialog, navigate to the `Advanced` tab, and
-    click `Environment Variables...`. Locate the `Path` variable and
-    click `Edit...`. In the `Edit environment variable` dialog, click `New` and
-    type the path to your PostgreSQL `pg_config` file. It should
-    be `C:\Program Files\PostgreSQL\14\bin\`. Click `OK` to save your changes.
-1.  Download the TimescaleDB installation `.zip` file from our
+
+    <highlight type="important">
+    You might need to add the `pg_config` file location to your path. In the Windows
+    Search tool, search for `system environment variables`. The path should be
+    `C:\Program Files\PostgreSQL\<version>\bin`.
+    </highlight>
+
+2.  Download the TimescaleDB installation `.zip` file from our
     [Windows releases][windows-releases].
-1.  Locate the downloaded file on your local file system, and extract the files.
-1.  In the extracted TimescaleDB directory, right-click the `setup.exe` file and
+3.  Locate the downloaded file on your local file system, and extract the files.
+4.  In the extracted TimescaleDB directory, right-click the `setup.exe` file and
     select `Run as Administrator` to start the installer.
 
 </procedure>

--- a/install/installation-windows.md
+++ b/install/installation-windows.md
@@ -47,10 +47,10 @@ current PostgreSQL installation, do not install TimescaleDB using this method.
     `C:\Program Files\PostgreSQL\<version>\bin`.
     </highlight>
 
-2.  Download the TimescaleDB installation `.zip` file from our
+1.  Download the TimescaleDB installation `.zip` file from our
     [Windows releases][windows-releases].
-3.  Locate the downloaded file on your local file system, and extract the files.
-4.  In the extracted TimescaleDB directory, right-click the `setup.exe` file and
+1.  Locate the downloaded file on your local file system, and extract the files.
+1.  In the extracted TimescaleDB directory, right-click the `setup.exe` file and
     select `Run as Administrator` to start the installer.
 
 </procedure>

